### PR TITLE
Inform about migrations downtime

### PR DESCRIPTION
**Pull request by FrNecas**
Preparing the announcement a bit in advance

### CODE REVIEW

**Review by lachmanfrantisek — APPROVED** [review]
Maybe, we can mention the deploy time in the text as well, WDYT? Is the `date`  enough?

**Comment by FrNecas** [comment]
Good idea, let's rather be explicit.

File: content/issues/2023-02-14-running-grouping-migrations.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+resolvedWhen:
+section: issue
+severity: down
+---
+
+We are updating our database schema to facilitate further development.
+As a part of this process, multiple database migrations will have to
+be run. The first step is grouping together related builds and test
+runs. The migrations will happen during the next deployment to production,
+on Tuesday February 14 at 2:00AM CET. This set of migrations is expected to
```
Comment (lachmanfrantisek) [inline content/issues/2023-02-14-running-grouping-migrations.md:21]: Users will get a Valentine's gift... :smile:

**Review by lachmanfrantisek — APPROVED** [review]

File: content/issues/2023-02-14-running-grouping-migrations.md (hunk 1 above)
Comment (FrNecas) [reply content/issues/2023-02-14-running-grouping-migrations.md:21]: Yeah, that was my immediate thought when I was checking the calendar - breaking the prod on Valentine's day :D

**Pull request merged**